packages/shared/src/lib/boot.ts

```diff
@@ -13,17 +13,24 @@ export type Boot = {
   visit: Visit;
   flags: IFlags;
   settings: RemoteSettings;
+  data?;
 };
 
 export type BootCacheData = Pick<
   Boot,
-  'user' | 'alerts' | 'settings' | 'flags'
->;
+  'user' | 'alerts' | 'settings' | 'flags' | 'data'
```

Comment:
What is the content of `data` property? I think we should change it to something much more explicit.